generators/generator-base.js

```diff
@@ -41,7 +41,7 @@ util.inherits(Generator, yeoman.Base);
  */
 Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enableTranslation) {
     try {
-        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.html';
+        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.component.html';
```

Comment:
Oh, so the idea is to have Angular 1 and 2 both running in JHipster 4.0 ? How can this work in both cases (NG1 / NG2) if we rename the file in NG2 but not in NG1 ? If we want to keep both working, that would mean to rename the files also in NG1, no ?